.gitignore

```diff
@@ -17,3 +17,6 @@ intro-to-R-tidyverse/04c-intro_to_tidyverse_exercise-part-2.nb.html
 
 # Snakemake files
 .snakemake
+
+# This is not a file that renv documentation recommends we track
+renv/settings.dcf
```

Comment:
Is there a reason this is here rather than in `renv/.gitignore`?

Reply:
Only for GitKraken reasons, can move